superset-frontend/src/explore/components/DataTableControl.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useMemo } from 'react';
+import { styled, t } from '@superset-ui/core';
+import { FormControl } from 'react-bootstrap';
+import Button from '../../components/Button';
+import RowCountLabel from './RowCountLabel';
+import {
+  applyFormattingToTabularData,
+  prepareCopyToClipboardTabularData,
+} from '../../utils/common';
+import CopyToClipboard from '../../components/CopyToClipboard';
+
+const CopyButton = styled(Button)`
+  padding: ${({ theme }) => theme.gridUnit / 2}px
```

Comment:
@rusackas do you think we should be doing division on `gridUnit`? I thought the intention was for this to be the smallest unit of sizing in the app

Reply:
Ideally we wouldn't divide. The gridUnit might become 5px one day, in which case we'd get into fractional-pixel-land, which will cause weird issues. The line below with `gridUnit * 2.5` would exhibit the same issues. How bad does this look with 1 gridunit on the top/bottom, 2 gridUnits on the sides? 

Alternatively, the Button supports a `buttonSize` prop, and `xs` or `s` values might look alright without overriding the padding at all

Reply:
That's how it looks with `buttonSize="xs"`. Paddings witht full gridUnits looked a bit awkward in my opinion.
![image](https://user-images.githubusercontent.com/15073128/100991358-330d4f00-3553-11eb-9eef-45192fb150aa.png)